docs/TEAM.md

```diff
@@ -3,6 +3,45 @@
 | Image                                                                      | Name                                                         | Description                                                                                                                                                                                                                                                                                                                                                                                                |
 | -------------------------------------------------------------------------- | ------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | <img src="./images/team/pranav.jpg" width="80px">                          | [Pranav Goel](https://github.com/pranavgoel29)               | I'm from India, Have basic knowledge of HTML, CSS, C++ and C, I'm currently learning Javascript.                                                                                                                                                                                                                                                                                                           |
+|                                                                                          | [Adewale Abiola](https://github.com/princehaybee)            | I am a Nigerian. I have basic knowledge of HTML,CSS and Javascript.                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
You copied this block twice (check from line 45).

You can see many members twice in this table.